google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/assembled_sql_objects_entry_factory.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.datacatalog_connectors.commons import prepare
+
+from google.datacatalog_connectors.rdbms.scrape import config_constants
+
+from google.datacatalog_connectors.rdbms.prepare.sql_objects\
+    import datacatalog_sql_objects_entry_factory,\
+    datacatalog_sql_objects_tag_factory
+
+
+class AssembledSQLObjectsEntryFactory:
+
+    def __init__(self, project_id, location_id, metadata_host_server,
+                 entry_group_id, sql_objects_config, tag_templates_dict):
+        self.__entry_group_id = entry_group_id
+        self.__sql_objects_config = sql_objects_config
+        self.__tag_templates_dict = tag_templates_dict
+        self.__datacatalog_entry_factory = \
+            datacatalog_sql_objects_entry_factory.\
+            DataCatalogSQLObjectsEntryFactory(
+                project_id,
+                location_id,
+                metadata_host_server,
+                self.__entry_group_id,
+                self.__sql_objects_config)
+        self.__datacatalog_tag_factory = datacatalog_sql_objects_tag_factory.\
+            DataCatalogSQLObjectsTagFactory(sql_objects_config)
+
+    def make_entries(self, sql_objects_metadata):
+        assembled_entries = []
+        if sql_objects_metadata:
```

Comment:
```suggestion
        if not sql_objects_metadata:
            return assembled_entries
```
A suggestion to avoid nesting code, especially due to the 79-char line length.